gui/default/syncthing/core/removeFolderDialogView.html

```diff
@@ -0,0 +1,15 @@
+<modal  id="remove-confirmation" status="danger" icon="exclamation-circle" heading="{{'Remove folder' | translate}}" large="no" closeable="yes">
+    <div class="modal-body">
+        <p ng-model="currentFolder.label" style=" overflow : hidden; text-overflow: ellipsis;">
+            <span translate>Are you sure you want to remove folder</span> {{currentFolder.label}}?
```

Comment:
This should probably be done with variable substitution, as not in all languages the folder name can go on the end of the sentence.